### PR TITLE
Fix for #64

### PR DESCRIPTION
**Pull request by Grimsbain**
Also added custom font and font size for pet frames.

### CODE REVIEW

File: Interface/AddOns/nMainbar/config.lua [hunk 1]
```diff
@@ -17,10 +17,13 @@ nMainbar.Config = {
 
         hotkeyFontsize = 18,
         hotkeyFont = 'Interface\\AddOns\\nMainbar\\media\\font.ttf',
+
+        petHotKeyFontsize = 14,
+        petHotKeyFont = 'Interface\\AddOns\\nMainbar\\media\\font.ttf',
```
Comment (renstrom) [inline Interface/AddOns/nMainbar/config.lua]: Not convinced this is a useful option, why would anyone have different fonts on the petbar and the main actionbar?

Comment (Grimsbain) [inline Interface/AddOns/nMainbar/config.lua]: I just swapped it out for hotkeyFont. I didn't feel it was really needed either but I wanted to leave the option open.

Comment (renstrom) [inline Interface/AddOns/nMainbar/config.lua]: More options isn't necessarily a good thing :)

**Comment by renstrom** [comment]
Great work with this, much appreciated!

**Comment by Grimsbain** [comment]
@renstrom I noticed that my fix only has the keybind showing red when you are out of range instead of the entire icon. Do you want me to change it back to the old way or keep it as is. It would just be two extra lines.

**Comment by renstrom** [comment]
Well I certainly would prefer if the entire icon turned red when out of range (like before). But if that can be achieved while still fixing #64, that would be awesome.

**Pull request merged**